.travis.yml

```diff
@@ -1,38 +1,25 @@
-if: branch = master OR tag IS present
+# It may be tempting to add parens around each individual clause in this expression, but Travis then builds pushes anyway
+if: branch = master OR branch =~ ^release/ OR tag IS present
 language: go
 go: 1.9
 sudo: true # give us 7.5GB and >2 bursted cores.
 git:
-  depth: false
+    depth: false
 before_install:
-    - export PULUMI_ROOT=/opt/pulumi
-    # Dep for Go dependency management.
-    - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-    # Gometalinter for good Go linting/hygiene.
-    - go get -v github.com/alecthomas/gometalinter
-    - gometalinter --install
-    # Node.js 6.10.2 for all JavaScript code (to match AWS Lambda).
-    - nvm install v6.10.2
-    # Install Yarn as per https://yarnpkg.com/lang/en/docs/install-ci/#travis-tab.
-    - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.2.1
-    - export PATH=$HOME/.yarn/bin:$PATH
-    # Install Twine so that we can publish Pip packages.
-    - pip install --upgrade --user twine==1.9.1
-    # Ensure that we can access Pulumi's private NPM and PyPI orgs.
-    - if [ ! -z "${NPM_TOKEN}" ]; then echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > ~/.npmrc; fi
-    # Install the AWS CLI so that we can publish the resulting release (if applicable) at the end.
-    - pip install --upgrade --user awscli
-    # We have some common scripts we use from the home and scripts repositories, so clone those repositories
     - git clone https://github.com/pulumi/scripts ${GOPATH}/src/github.com/pulumi/scripts
+    - git -C ${GOPATH}/src/github.com/pulumi/scripts checkout -b origin/ellismg/update-before-install-pandoc
```

Comment:
I feel like this is different than the other repos I've just looked at.  Is this intentional?

Reply:
I was using it to test something.  I have removed it and now this looks like the other branches.